### ISSUE TITLE
Image missing text alternative: collection poster images

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5338]
Sites identified: MCO, IJCCR
Page(s) identified: 

- Collections index page (poster image for collections, including the default folder image)

WCAG compliance level: A

Description:

All non-decorative images must have a text alternative (also known as "alt text").

Text alternatives are used to describe images to people who are unable to see them. This includes people who use screen readers to access the web.

The text alternative should describe the image within the context of the page — letting visitors know what the image means and why it was included.

WCAG 2.1: [Understanding Success Criterion 1.1.1: Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
Siteimprove Alfa: [Technical documentation for rule SIA-R2](https://alfa.siteimprove.com/rules/sia-r2)

See more details at [SiteImprove example](https://my2.siteimprove.com/Inspector/1167098/A11Y/Page?pageId=37959502261&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r2/failed/emEvclpQL0Y=;ODNiZjk3OWQ=)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBa0pnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--070ed5a911674a394ad5da570aea2953872c25ca/image.png)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBazVnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--30e056dac2231e207b43142af0ac06d6236b00be/image.png)

QUESTION: are posters non-decorative images? For this purpose, yes. We have an existing issue to allow users to set custom text for thumbnails; this could also apply to these poster images if collection staff wanted to add description to these images that represent the collection.

# Done Looks Like
- [ ] Poster images are indicated as decorative images via attribute

**Comment by joncameron** [comment]
We could implement some stock text for classification of some images presented; collection poster images can be described as representing the collection, rather than providing a description of what the image presents.
